pom.xml

```diff
@@ -2718,7 +2710,6 @@
                     <bannedDependencies>
                       <excludes combine.children="append">
                         <exclude>*:*_2.11</exclude>
-                        <exclude>*:*_2.10</exclude>
```

Comment:
@dbtsai sorry for the late idea here -- this isn't essential for the change, and you don't have to make it here -- but I thought of a better way. Really we want the default `maven-enforcer-plugin` config above to exclude _2.10 and _2.11 dependencies, and remove everything from the `scala-2.12` profile (or else, one still has to enable the profile to get all Scala 2.12 config). Then, move this `maven-enforcer-plugin` config to the `scala-2.11` profile. That copy should only exclude _2.10 dependencies. However to make sure Maven doesn't also add that to the _2.11 exclusion rule in the parent, the `combine.children="append"` attribute here can become `combine.self="override"`. That should get the desired effects.

Reply:
Thanks for the suggestion, and I agree this will make the default scala 2.12 profile cleaner.